### PR TITLE
Add functionality for playlist screen, refactored screen to look like…

### PR DESCRIPTION
**Pull request by NathanDVT**
- [X] Implemented UI screen layout for Playlist screen
- [X] Implemented UI screen layout for Playlist detail screen
- [X] Add design constants for view, image and button
- [X] Integrated basic functionality to UI screen
Screenshot 2020-04-23 at 17 04 09
Screenshot 2020-04-23 at 17 03 43

### CODE REVIEW

File: realwork/realwork/DesignConstants/ViewDesigns.swift [hunk 1]
```diff
@@ -118,3 +118,22 @@ class InversedPrimaryShadowView: ViewInversePrimaryGradient {
...
+        setup()
+    }
+
+    private func setup() {
+        self.layer.cornerRadius = 10
+        self.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner,
+                                    .layerMinXMaxYCorner, .layerMaxXMaxYCorner]
+    }
+}
+
```
Comment (NathanDVT) [inline realwork/realwork/DesignConstants/ViewDesigns.swift:139]: Codacy Issue found: [Files should have a single trailing newline.](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5409503)

**Comment by NathanDVT** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- realwork/realwork/PlaylistBoard/PlaylistBoardVC.swift  1
- realwork/realwork/DesignConstants/ViewDesigns.swift  5
- realwork/realwork/PlaylistBoard/PlaylistDetailViewController.swift  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5409503&bid=17735908)

**Comment by NathanDVT** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- realwork/realwork/PlaylistBoard/PlaylistBoardVC.swift  1
- realwork/realwork/DesignConstants/ViewDesigns.swift  5
- realwork/realwork/PlaylistBoard/PlaylistDetailViewController.swift  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5409503&bid=17735908)

File: realwork/realwork/PlaylistBoard/PlaylistDetailViewController.swift [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+    /*
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        // Get the new view controller using segue.destination.
+        // Pass the selected object to the new view controller.
+    }
+    */
```
Comment (JancoDVTDev) [inline realwork/realwork/PlaylistBoard/PlaylistDetailViewController.swift:42]: Can remove this commented code

File: realwork/realwork/PlaylistBoard/PlaylistDetailViewController.swift [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "PlaylistCell",
+                                                       for: indexPath) as? PlaylistTableViewCell else {
+            return UITableViewCell()
+        }
+        cell.nameUILabel.text = "\(selectedPlaylist!.basicSongItems[indexPath.row].songTitle)"
+        cell.numSongsUILabel.text = "Number of songs: \(selectedPlaylist!.basicSongItems[indexPath.row].songTitle)"
+        return cell
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
```
Comment (JancoDVTDev) [inline realwork/realwork/PlaylistBoard/PlaylistDetailViewController.swift:64]: Remove Function of not in use

**Review by JancoDVTDev — APPROVED** [review]
Looks good

**Pull request merged**